src/app.ts

```diff
@@ -0,0 +1,25 @@
+import * as express from 'express';
+import * as swaggerUI from 'swagger-ui-express';
+import * as path from 'path';
+import * as YAML from 'yamljs';
```

Comment:
Consider not to use syntax like `import * as something` for default import

Reply:
Noted. Amended